src/constants/tokenLists/ethereum.tokenlist.js

```diff
@@ -48,6 +48,27 @@ const ETHEREUM_TOKEN_LIST = {
     decimals: 18,
     logoURI: 'https://drive.google.com/uc?export=download&id=16PVXI1Da5P27cywWiyqrcV-Q17u1aXsi',
   },
+  '0x0f17bc9a994b87b5225cfb6a2cd4d667adb4f20b': {
+    address: '0x0f17bc9a994b87b5225cfb6a2cd4d667adb4f20b',
+    symbol: 'jEUR',
+    name: 'Jarvis Synthetic Euro',
+    decimals: 18,
+    logoURI: 'https://i.imgur.com/ZvP634G'
```

Comment:
Please use the direct link for logoURI
For example, https://i.imgur.com/ZvP634G.png

The same applies to other places